schemas/event.yml

```diff
@@ -99,45 +220,108 @@
+        - name: success
+          description: >
+            Indicates that this event describes a successful result.  A common example is
+            `event.category:file AND event.type:create AND event.outcome:success`
+            to indicate that a file was successfully created.
+        - name: unknown
+          description: >
+            Indicates that this event describes only an attempt for which the result
+            is unknown. For example, if the event contains information only about a
+            request in an entity transaction that usually results in a response,
+            populating `event.outcome:unknown` is appropriate.
 
     - name: type
       level: core
       type: keyword
-      short: Reserved for future usage.
+      short: Event type (Categorization Field)
       description: >
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-
-      accepted_values:
-      - name: start
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-
-      - name: end
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
+        This is one of four ECS Categorization Fields, and indicates the
+        third level in the ECS category hierarchy.
+
+        `event.type` represents a categorization "sub-bucket" that, when used along with
+        the `event.category` field values, enables filtering events down to a level
+        appropriate for single visualization.
+      allowed_values:
+        - name: access
+          description: >
+            The access event type is used for the subset of events within a category
+            that indicate that something was accessed.
+            Common examples include `event.category:database AND event.type:access`,
+            or `event.category:file AND event.type:access`.
+            Note for file access, both directory listings and file opens should be included
+            in this subcategory. You can further distinguish access operations using the ECS
+            `event.action` field.
+        - name: change
+          description: >
+            The change event type is used for the subset of events within a category
+            that indicate that something has changed. If semantics best describe an
+            event as modified, then include them in this subcategory.
+            Common examples include `event.category:process AND event.type:change`,
+            and `event.category:file AND event.type:change`.
+            You can further distinguish change operations using the ECS `event.action` field.
+        - name: creation
+          description: >
+            The "creation" event type is used for the subset of events within a category
```

Comment:
should we quote the other values like this? (i.e. "access", "change", "deletion", etc.) Also do we need the sentence structure to be consistent with whatever we do for `event.category` values?

Reply:
I vote for consistency.  In this case, let's just remove the quotes.